pegasus/helper_modules/dashboard.rb

```diff
@@ -0,0 +1,93 @@
+#
+# A set of utilities that provide access to the current user, their
+# authentication state, and other DASHBOARD_DB access.
+#
+module Dashboard
+
+  # Get the dashboard database
+  # The static constant is wrapped here so it's easy to stub for tests.
+  # @returns [Database]
+  def self.db
+    DASHBOARD_DB
+  end
+
+
+  def self.admin?(user_id)
+    !!db[:users].where(id: user_id, admin: true).first
```

Comment:
Unfortunately, the `exists?` method you propose doesn't exist on [Sequel::Dataset](http://sequel.jeremyevans.net/rdoc/classes/Sequel/Dataset.html).  The closest thing is [this `exists` method`](http://sequel.jeremyevans.net/rdoc/classes/Sequel/Dataset.html#method-i-exists) which takes no arguments and returns an "EXISTS" clause string.  I think that would just expand this line:

```ruby
# Generates something like
# SELECT EXISTS(SELECT * FROM users WHERE id=:user_id, admin=true)
# which still needs to be resolved by calling `first` (or `all`)
!!db[:users].where(id: user_id, admin: true).exists.first
```